Easydict/Swift/View/SettingView/Tabs/ServiceConfigurationView/TextEditorCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TextEditorCell.swift
+//  Easydict
+//
+//  Created by tisfeng on 2024/5/27.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+// MARK: - TextEditorCell
+
+struct TextEditorCell: View {
+    // MARK: Internal
+
+    let title: LocalizedStringKey
+    @Binding var text: String
+    let placeholder: LocalizedStringKey
+
+    var body: some View {
+        HStack(alignment: .center, spacing: 20) {
+            Text(title)
+
+            TextEditorWithPlaceholder(text: $text, placeholder: placeholder, alignment: .topTrailing)
```

Comment:
It looks strange when the text is empty.
![CleanShot 2024-05-28 at 10 38 18@2x](https://github.com/tisfeng/Easydict/assets/26460192/846edbbb-f067-407f-8541-3855b98c0b7d)

Reply:
I try change `ZStack` alignment type to `.topLeading` 
```suggestion
            TextEditorWithPlaceholder(text: $text, placeholder: placeholder, alignment: .topLeading)
```